diff -ubr express-1.5.1-src/src/CMakeLists.txt express-1.5.1-src-dynamic-lib-patch/src/CMakeLists.txt
--- express-1.5.1-src/src/CMakeLists.txt	2013-12-08 14:09:18.000000000 -0500
+++ express-1.5.1-src-dynamic-lib-patch/src/CMakeLists.txt	2015-07-19 16:44:08.546696666 -0400
@@ -10,17 +10,17 @@
 set(LIBRARIES ${Boost_LIBRARIES} ${ZLIB_LIBRARIES})
 
 if (GPERFTOOLS_TCMALLOC) 
-   set(LIBRARIES ${LIBRARIES} "libtcmalloc_minimal.a")
+   set(LIBRARIES ${LIBRARIES} "tcmalloc_minimal")
 endif (GPERFTOOLS_TCMALLOC)
 
 if(WIN32)
-  set(LIBRARIES ${LIBRARIES} "${CMAKE_CURRENT_SOURCE_DIR}/../bamtools/lib/libbamtools.lib" "${CMAKE_CURRENT_SOURCE_DIR}/../win_build/zlibd.lib")
+  set(LIBRARIES ${LIBRARIES} "bamtools" "zlibd")
 else(WIN32)
-  set(LIBRARIES ${LIBRARIES} "${CMAKE_CURRENT_SOURCE_DIR}/../bamtools/lib/libbamtools.a" "pthread")
+  set(LIBRARIES ${LIBRARIES} "bamtools" "pthread")
 endif(WIN32)
 
 if (PROTOBUF_FOUND)
-  set(LIBRARIES ${LIBRARIES} "libprotobuf.a")
+  set(LIBRARIES ${LIBRARIES} "protobuf")
 endif(PROTOBUF_FOUND)
 
 target_link_libraries(express ${LIBRARIES})
