diff --git a/Makefile b/Makefile
index 471242d..ea7e979 100644
--- a/Makefile
+++ b/Makefile
@@ -53,6 +53,8 @@ bindir      = $(exec_prefix)/bin
 datarootdir = $(prefix)/share
 mandir      = $(datarootdir)/man
 man1dir     = $(mandir)/man1
+libdir      = $(prefix)/lib
+includedir  = $(prefix)/include/bam
 
 # Installation location for $(MISC_PROGRAMS) and $(MISC_SCRIPTS)
 misc_bindir = $(bindir)
@@ -297,11 +299,13 @@ misc/maq2sam-long.o: misc/maq2sam.c config.h version.h
 
 
 install: $(PROGRAMS) $(MISC_PROGRAMS)
-	$(INSTALL_DIR) $(DESTDIR)$(bindir) $(DESTDIR)$(misc_bindir) $(DESTDIR)$(man1dir)
+	$(INSTALL_DIR) $(DESTDIR)$(bindir) $(DESTDIR)$(misc_bindir) $(DESTDIR)$(man1dir) $(DESTDIR)$(libdir) $(DESTDIR)$(includedir)
 	$(INSTALL_PROGRAM) $(PROGRAMS) $(DESTDIR)$(bindir)
 	$(INSTALL_PROGRAM) $(MISC_PROGRAMS) $(DESTDIR)$(misc_bindir)
 	$(INSTALL_SCRIPT) $(MISC_SCRIPTS) $(DESTDIR)$(misc_bindir)
 	$(INSTALL_MAN) samtools.1 misc/wgsim.1 $(DESTDIR)$(man1dir)
+	$(INSTALL_DATA) libbam.a $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) *.h $(DESTDIR)$(includedir)
 
 
 testclean:
